docs/user-documentation/2xfor1x.md

```diff
@@ -60,7 +60,7 @@ In Fedora 4:
 * Ingest occurs asynchronously soon after. 
 
 ###Collections
-Because objects in Fedora 3 were stored in a flat graph structure instead of a hierarchy, what were presented as collection in Islandora 7.x-1.x were actually objects on the same level as their child objects, with the 'container' or 'folder' aspect of them being a fiction for display created by the relationships between the objects. In Fedora 4, resources do have a true hierarchical structure and must have a `fedora:hasParent` relationship to know where they belong in a given repository. Indeed, to migrate objects over from Fedora 3 to Fedora 4, parents must arrive before their children. In its current incarnation, all objects in a 7.x-2.x repository have the Fedora root as a parent and relationships are managed via the ```pcdm:hasMember``` predicate. 
+Because objects in Fedora 3 were stored in a flat graph structure instead of a hierarchy, what were presented as collections in Islandora 7.x-1.x were actually objects on the same level as their child objects, with the 'container' or 'folder' aspect of them being a fiction for display created by the relationships between the objects. In Fedora 4, resources do have a true hierarchical structure and must have a `fedora:hasParent` relationship to know where they belong in a given repository. Indeed, to migrate objects over from Fedora 3 to Fedora 4, parents must arrive before their children. In its current incarnation, all objects in a 7.x-2.x repository have the Fedora root as a parent and relationships are managed via the ```pcdm:hasMember``` predicate. 
```

Comment:
I'm just catching up to this, so a  couple of questions that might go outside the scope of this PR:

1. "In Fedora 4, resources do have a true hierarchical structure and must have a `fedora:hasParent` relationship to know where they belong in a given repository." I don't understand the meaning of this at all. LDP containment isn't signified by that predicate and LDP containment is (only currently, only for the community impl) the necessary condition for a resource to be located. The spec doesn't mention that predicate at all, nor does it demand LDP containment for a resource at all, and sometime this spring the community impl will meet that. So the language here maybe could get a little looser?

2. Using @ruebot's edit "In its current incarnation, all objects in an Islandora CLAW repository have the Fedora root as a parent"-- wh-what! Seriously? All resources are direct children of the root? I think there must be some wording unclarity here...?